lms/celery.py

```diff
@@ -21,3 +21,42 @@
 # pickle the object when using Windows.
 APP.config_from_object('django.conf:settings')
 APP.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
+
+# Some tasks require settings defined on non-lms workers, or they will crash.
+# This dict defines those tasks, and the worker environment they should be routed towards.
+ALTERNATE_ENV_TASKS = {
+}
+
+
+class Router(object):
```

Comment:
Why duplicate the implementation of this Router class?  It's nicely generic enough to be reused in both LMS and CMS.  So that only the `ALTERNATE_ENV_TASKS` needs to be defined in each.
You can create a base `Router` class that is shared by both.